apollo-compiler/src/test/graphql/com/example/java_beans_semantic_naming/type/Episode.kt

```diff
@@ -5,15 +5,16 @@
 //
 package com.example.java_beans_semantic_naming.type
 
+import com.apollographql.apollo.api.EnumValue
 import kotlin.Deprecated
 import kotlin.String
 
 /**
  * The episodes in the Star Wars trilogy
  */
 enum class Episode(
-  val rawValue: String
-) {
+  override val rawValue: String
+) : EnumValue {
```

Comment:
Paranoid checks: I think this is a backward compatible change but since it touches regular enums and not only sealed classes this has the potentially to impact a lot more users. Are we missing possibly missing a weird edge case?

Reply:
I don't see any 🤔 One edge case would be using jar with generated old classes (no interface) with new runtime, but it should work (just the check won't trigger). For enum classes behavior doesn't change at all, as from what I see both raw value and enum name always match (so cache won't break when updating app with new generated classes). 

I can of course revert this if you find some problem or just feel I should only fix the bug with sealed classes and let enum classes as is. It just felt weird leaving actual enum classes without having them implement `EnumValue` interface 🙂

Reply:
Yep, I think it's safe too 👍 and way more uniform like this 👍 .